pkg/antctl/command_definition_test.go

```diff
@@ -17,19 +17,126 @@ package antctl
 import (
 	"bytes"
 	"encoding/json"
+	"fmt"
 	"io"
 	"reflect"
 	"strings"
 	"testing"
 
 	"github.com/spf13/cobra"
 	"github.com/stretchr/testify/assert"
+
+	"github.com/vmware-tanzu/antrea/pkg/antctl/transform/addressgroup"
+	"github.com/vmware-tanzu/antrea/pkg/antctl/transform/appliedtogroup"
+	"github.com/vmware-tanzu/antrea/pkg/antctl/transform/common"
+	"github.com/vmware-tanzu/antrea/pkg/antctl/transform/networkpolicy"
+	"github.com/vmware-tanzu/antrea/pkg/antctl/transform/rule"
+	networkingv1beta1 "github.com/vmware-tanzu/antrea/pkg/apis/networking/v1beta1"
 )
 
 type Foobar struct {
 	Foo string
 }
 
+func TestCommandList_tableOutputForGetCommands(t *testing.T) {
+	for _, tc := range []struct {
+		name            string
+		rawResponseData interface{}
+		expected        string
+	}{
+		{
+			name:            "StructureData-NetworkPolicy-List-EmptyRespCase",
+			rawResponseData: []networkpolicy.Response{},
+			expected:        "\n",
+		},
+		{
+			name: "StructureData-NetworkPolicy-List-HasSummary-RandomFieldOrder",
+			rawResponseData: []networkpolicy.Response{
+				{
+					Rules: []rule.Response{
+						{
+							Direction: "In",
+							Services:  nil,
+						},
+						{
+							Direction: "In",
+							Services:  nil,
+						},
+					},
+					AppliedToGroups: []string{"32ef631b-6817-5a18-86eb-93f4abf0467c"},
+					Name:            "GroupName1",
+				},
+				{
+					Name:            "GroupName2",
+					AppliedToGroups: []string{"32ef631b-6817-5a18-86eb-93f4abf0467c", "c4c59cfe-9160-5de5-a85b-01a58d11963e"},
+					Rules: []rule.Response{
+						{
+							Direction: "In",
+							Services:  nil,
+						},
+					},
+				},
+			},
+			expected: "+------------+--------------------------------------+-------+\n" +
```

Comment:
you may want to look into using raw literals for these expected outputs

Reply:
> you may want to look into using raw literals for these expected outputs

OK, I will make some modifies ASAP.